containers.go

```diff
@@ -52,24 +52,28 @@ func (c *Container) Start(ctx context.Context) (status string, err error) {
 	return status, c.client.Post(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/status/start", c.Node, c.VMID), nil, &status)
 }
 
-func (c *Container) Stop(ctx context.Context) (status *ContainerStatus, err error) {
+func (c *Container) Stop(ctx context.Context) (status string, err error) {
```

Comment:
oh did we really add this before the the data field was being auto-parsed... goodness that was a long time ago. great fix and probably can remove the ContainerStatus type too right? it's at this point useless

Reply:
Actually when I checked the API doc realized it turns the UPID as a string so I don't think we need something like containerStatus. IMO returning the UPID as it is might not be the correct way to do it so we can convert them to tasks. Here is an example of start but any status command returns the same object:

`root@lowtower:~# pvesh create /nodes/lowtower/lxc/150/status/start
UPID:lowtower:0035154F:068C3DB0:65E99E9D:vzstart:150:root@pam:`

So what I'm offering is converting that one to 

`func (c *Container) Stop(ctx context.Context) (task *Task, err error) {
	var upid UPID
	if err := c.client.Post(ctx, fmt.Sprintf("/nodes/%s/lxc/%d/status/stop", c.Node, c.VMID), nil, &upid); err != nil {
		return nil, err
	}
	return NewTask(upid, c.client), nil
}`

Reply:
i agree to that change, let's do that. UPID and Tasks didnt exist when those funcs were initially wrote so let's go that route.